daemon/list.go

```diff
@@ -265,7 +265,7 @@ func (daemon *Daemon) foldFilter(view container.View, config *types.ContainerLis
 
 	err = psFilters.WalkValues("status", func(value string) error {
 		if !container.IsValidStateString(value) {
-			return fmt.Errorf("Unrecognised filter value for status: %s", value)
+			return validationError{errors.Errorf("Unrecognised filter value for status: %s", value)}
```

Comment:
Wondering why there's a special type for `invalidFilter`, but `error.Errorf()` here